data/db_demo.yaml

```diff
@@ -317,7 +317,8 @@ instrument:
     field_data:
       file: ../data/ZTF_Fields.csv
     field_region:
-      file: ../data/ZTF_Square_Region.reg
+      file: ../data/ZTF_Region.reg
```

Comment:
We probably still want the Square one for the demo data due to its speed.

Reply:
Do we want the square in production or only in the testing? I think we want to make it as slow as possible to make sure the new code is fast enough even with the full region.

Reply:
I think only in testing. We can run the real one in production.